internal/store/owners.go

```diff
@@ -9,14 +9,18 @@ import (
 )
 
 type OwnerCreator interface {
-	CreateOwner(*cm.User, string) (*cm.CreateUserResponse, error)
+	CreateOwner(*cm.User, []string) (*cm.CreateUserResponse, error)
 }
 
 // CreateOwner encapsulate the logic to manage different cloud providers
-func (s *Store) CreateOwner(u *cm.User, IPs string) (*cm.CreateUserResponse, error) {
+func (s *Store) CreateOwner(u *cm.User, IPs []string) (*cm.CreateUserResponse, error) {
 	switch s.service {
 	case config.OpsManagerService:
-		opts := &cm.WhitelistOpts{Whitelist: IPs}
+		var opts *cm.WhitelistOpts
+		if len(IPs) > 0 {
+			opts = &cm.WhitelistOpts{Whitelist: IPs}
```

Comment:
[q] Do we have a test that verifies the `CreateFirstUser` functionality? The calls are pretty simple right now, but as we add more parameters it would be good to see any validation that's necessary on the inputs it done correctly.

Reply:
We do have a test, adding a scenario with and without the param